pkg/aws/services/vpclattice.go

```diff
@@ -118,9 +120,11 @@ func NewDefaultLattice(sess *session.Session, region string) *defaultLattice {
 		endpoint = latticeEndpoint
 	}
 
-	latticeSess = vpclattice.New(sess, aws.NewConfig().WithRegion(region).WithEndpoint(endpoint).WithMaxRetries(20))
+	latticeSess := vpclattice.New(sess, aws.NewConfig().WithRegion(region).WithEndpoint(endpoint).WithMaxRetries(20))
```

Comment:
what do you think about 20 retries and it's impact on throttling/performance?

Reply:
I have never observed backoffs getting more than 2~3 on manual tests, so it is hard for me to think that it is affecting anything. we can definitely revisit this later though.